data.sql

```diff
@@ -14,3 +14,77 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
     ('Boarmon', '2005-06-07', 7, TRUE, 20.4, 'unspecified'),
     ('Blossom', '1998-10-13', 3, TRUE, 17.00, 'unspecified'),
     ('Ditto', '2022-05-14', 4, TRUE, 22.00, 'unspecified');
+
+    
+/* day 3 of the query multiple tables*/
+-- Create the owners table
+CREATE TABLE owners (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    full_name VARCHAR(255),
+    age INT
+);
+
+-- Insert data into the owners table
+
+INSERT INTO owners (full_name, age) VALUES
+    ('Sam Smith', 34),
+    ('Jennifer Orwell', 19),
+    ('Bob', 45),
+    ('Melody Pond', 77),
+    ('Dean Winchester', 14),
+    ('Jodie Whittaker', 38);
+
+-- Create the species table
+CREATE TABLE species (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255)
+);
```

Comment:
👍 Good progress on adding the indexes! However, there's a small adjustment to make to align with the curriculum guidelines.

- [ ] Currently, your `species ` creation operation is inside the `datas.sql` file. According to the [curriculum-databases-projects-template](https://github.com/microverseinc/curriculum-template-databases#:~:text=This%20repository%20includes,Use%20them%20responsibly!), `CREATE` operations should be inside the `schema.sql` file. Moreover, you already have the same operation inside the `schema.sql` file. I will recommend you remove this operation to align with the requirement.

By organizing your SQL scripts this way, you'll ensure that your project structure aligns with the curriculum guidelines. Keep up the great work! 👍📜

Reply:
Thank you for your review i will look into that